lib/Mojolicious/Plugin/AssetPack/Pipe/Fetch.pm

```diff
@@ -3,8 +3,26 @@ use Mojo::Base 'Mojolicious::Plugin::AssetPack::Pipe';
 use Mojolicious::Plugin::AssetPack::Util qw(diag DEBUG);
 use Mojo::URL;
 
-# Only made public for quick fixes. Subject for change
-our $URL_RE = qr{url\((['"]{0,1})(.*?)\1\)};
+our %formats = (
```

Comment:
Please use capital letters for "global" variables, like `%FORMATS` or `%URL_RE`.